packages/core/src/mechanic-validation.js

```diff
@@ -197,16 +196,24 @@ const prepareValues = (inputs, settings, baseValues) => {
     }
   });
 
-  // Scale down to fit if width and height are inputs
-  if (baseValues.scaleToFit && values.width && values.height) {
-    const ratioWidth = baseValues.scaleToFit.width ? baseValues.scaleToFit.width / values.width : 1;
-    const ratioHeight = baseValues.scaleToFit.height
-      ? baseValues.scaleToFit.height / values.height
-      : 1;
-    if (ratioWidth < 1 || ratioHeight < 1) {
-      const ratio = ratioWidth < ratioHeight ? ratioWidth : ratioHeight;
-      values.width = Math.floor(values.width * ratio);
-      values.height = Math.floor(values.height * ratio);
+  // Add ratio and original values if width and height are inputs
+  if (values.width && values.height) {
+    values.widthOriginal = values.width
+    values.heightOriginal = values.height
+    values.ratio = 1
```

Comment:
I'm thinking that maybe the names for this could be different, just so that there's no name collision with the inputs defined by the user. Even though `widthOriginal` and `heightOriginal` don't seem like go-to names, `ratio` does and I've even used it. 

Maybe prepending an underscore to all "added" values from Mechanic could be the way. So `_width`, `_height` and `_ratio`. I'm not totally convinced about that specific naming convention, so I would love to hear your thoughts on it!

Reply:
I think it's a good idea to prefix it - and an underscore sounds fine. I've changed it in commit [19211bc](https://github.com/designsystemsinternational/mechanic/pull/113/commits/19211bce4ad9cd54e93590b69c124a9581ea4599).